### PR TITLE
config: update the default value of `feedback-probability`

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added or deleted? (Required)

The feature`feedback-probability` has been disabled by default since v4.0.9 because its behavior is unstable (see https://github.com/pingcap/tidb/pull/21923). So I update the default value of it to `0` and modify some descriptions.

After this PR is cherry-picked to the release-4.0 branch, I will add the information that the feature`feedback-probability` is disabled from the patch version v4.0.9.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: statistics.md [hunk 1]
```diff
@@ -147,11 +147,11 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 当某个表 `tbl` 的修改行数与总行数的比值大于 `tidb_auto_analyze_ratio`，并且当前时间在 `tidb_auto_analyze_start_time` 和 `tidb_auto_analyze_end_time` 之间时，TiDB 会在后台执行 `ANALYZE TABLE tbl` 语句自动更新这个表的统计信息。
 
-在查询语句执行时，TiDB 会以 `feedback-probability` 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。可通过配置文件修改 `feedback-probability`，其默认值是 `0.05`。设置成 `0.0` 可以关闭这个功能。
+在查询语句执行时，TiDB 会以 `feedback-probability` 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。
```
Comment (yikeke) [inline statistics.md:150]: `feedback-probability` 上附个配置文件的链接吧

Comment (yikeke) [reply statistics.md]: 不过，这个配置项默认关闭了，这句话就不应该写在这了吧。

Comment (CharLotteiu) [reply statistics.md]: Updated in https://github.com/pingcap/docs-cn/pull/6113/commits/ee784be9149e7d4062a8926fff8718d0b89e2a19.

File: statistics.md [hunk 2]
```diff
@@ -147,11 +147,11 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 当某个表 `tbl` 的修改行数与总行数的比值大于 `tidb_auto_analyze_ratio`，并且当前时间在 `tidb_auto_analyze_start_time` 和 `tidb_auto_analyze_end_time` 之间时，TiDB 会在后台执行 `ANALYZE TABLE tbl` 语句自动更新这个表的统计信息。
 
-在查询语句执行时，TiDB 会以 `feedback-probability` 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。可通过配置文件修改 `feedback-probability`，其默认值是 `0.05`。设置成 `0.0` 可以关闭这个功能。
+在查询语句执行时，TiDB 会以 [`feedback-probability`](/tidb-configuration-file.md#feedback-probability) 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。
```
Comment (yikeke) [inline statistics.md:150]: 这个配置项默认关闭了，也就是这个功能用户默认感知不到？那这句话就不应该写在这了吧。

Comment (winoros) [reply statistics.md:150]: 或者这里提一下是默认关闭的这样？

Comment (yikeke) [reply statistics.md:150]: 现在是不建议使用这个功能？不建议使用的功能就不用记在文档里了，降低用户心智成本

Comment (winoros) [reply statistics.md:150]: 是的 目前不建议使用

Comment (winoros) [reply statistics.md:150]: 不过这个地方主要是之前的版本是有的。
3.0 4.0 文档里有，5.0里反而没有也比较奇怪？

**Comment by TomShawn** [comment]
@yikeke @wangtianyi2004 Any update?

**Comment by yikeke** [comment]
请 @CharLotteiu 推进下吧

**Comment by CharLotteiu** [comment]
@winoros @yikeke Please take a look again. Thx!

File: statistics.md [hunk 3]
```diff
@@ -147,11 +147,7 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 当某个表 `tbl` 的修改行数与总行数的比值大于 `tidb_auto_analyze_ratio`，并且当前时间在 `tidb_auto_analyze_start_time` 和 `tidb_auto_analyze_end_time` 之间时，TiDB 会在后台执行 `ANALYZE TABLE tbl` 语句自动更新这个表的统计信息。
 
-在查询语句执行时，TiDB 会以 `feedback-probability` 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。可通过配置文件修改 `feedback-probability`，其默认值是 `0.05`。设置成 `0.0` 可以关闭这个功能。
-
-> **注意：**
->
-> 在配置文件中如果将 `feedback-probability` 设置为 `0` 会导致设置失败并报错。需要设置成 `0.0` 才可以关闭 `feedback-probability`。
+在 v5.0 之前，查询语句执行时，TiDB 会以 [`feedback-probability`](/tidb-configuration-file.md#feedback-probability) 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。**在 v5.0 中，该功能默认关闭，暂不建议开启此功能。**
```
Comment (yikeke) [inline statistics.md:150]:
```suggestion
在 v5.0 版本之前，执行查询语句时，TiDB 会以 [`feedback-probability`](/tidb-configuration-file.md#feedback-probability) 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。**对于 v5.0 版本，该功能默认关闭，暂不建议开启此功能。**
```

**Review by yikeke — APPROVED** [review]
rest lgtm

**Review by winoros — APPROVED** [review]

**Pull request merged**